src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -235,21 +250,4 @@
     <Reference Include="System.Threading.Tasks" />
     <Reference Include="System.Threading.Tasks.Extensions" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
We should keep this separate item group for now since we `JsonNode` APIs haven't gone through final API review and approval yet.

Reply:
What difference makes to have them on a separate `ItemGroup`?

Reply:
We want to keep them separate for now so if we need to remove it for a preview release, it is easier to do so.

Reply:
I assume we'd have a condition applied to the group at that time.